src/App.tsx

```diff
@@ -1,11 +1,25 @@
-import React from 'react';
-import Mainform from './components/MainForm/Mainform';
-import './App.css';
+import React from "react";
+import Mainform from "components/MainForm/Mainform";
+import Splash from "components/LoadingScreen/Splash";
+import { BrowserRouter, Route } from "react-router-dom";
+import "./App.css";
+import { Provider } from "react-redux";
+import store from "store/reducers/mainForm";
+
 
 function App() {
   return (
     <div className="App">
-     <Mainform/>
+      <Provider store={store}>
```

Comment:
@ZainShahbaz Please use Prettier and eslint to format/align your code!!
btw good work here

Reply:
okay